taskchampion/CONTRIBUTING.md

```diff
@@ -24,11 +24,8 @@ Other ideas;
 
 TaskChampion is a typical Rust application.
 To work on TaskChampion, you'll need to [install the latest version of Rust](https://www.rust-lang.org/tools/install).
-Once you've done that, run `cargo build` at the top level of this repository to build the binaries.
```

Comment:
Would it make sense to say here that the build is now typically invoked through cmake?

Reply:
Once this is a separate project, that won't make sense. This PR is just trying to remove sync-server, with more to come in #3209.